README.md

```diff
@@ -284,13 +284,13 @@ Once you are done with the development, you can deploy the service using any of
    $ ballerina build inventory_services
 ```
 
-- Once the balx files are created inside the target folder, you can run them with the following commands. 
+- Once the balx files are created inside the target folder, navigate to target folder and then you can run them with the following commands. 
```

Comment:
Change to: 'Once the .balx files are created inside the target folder, navigate to the target folder and run them with the following commands.'